doc/sphinx/methods.rst

```diff
@@ -361,6 +361,60 @@ To calculate Bader charges from a cube file, a ``Volume`` object should be prepa
     m.calculate()
 
 
+.. index::
+    single: methods; DDEC6
+    
+DDEC6
+----------------
+
+DDEC6 charges are stockholder-like charge partitioning method introduced in `[Manz2016] <https://doi.org/10.1039/C6RA04656H>`_
+
+.. code-block:: python
+
+    from cclib.method import Bader
```

Comment:
Bader -> DDEC6?

Reply:
It looks like I've mixed up PRs while writing up the docs. I've removed this part -- the docs are in a separate PR. Thanks for letting me know!

Reply:
nvm i didnt see the other PR